docs/index.html

```diff
@@ -106,7 +107,7 @@ <h2>Software development</h2>
             <h2>It's affordable</h2>
             <span>
               Joy of Coding is a non-profit event. With the help of our sponsors we keep registration fees low to
-              provide the best value-for-money conference of 2023. Our very-early bird tickets sell at only EUR 128,-
+              provide the best value-for-money conference of 2024. Our very-early bird tickets sell at only EUR 128,-
```

Comment:
Are we sure about this price?

Reply:
Good catch, zinnetje over de prijs gewoon weg gehaald.